extensions/wikia/Track/Track.php

```diff
@@ -65,6 +65,7 @@ private static function getViewJS ($param=null) {
 		window.beacon_id = result[1];
 	} else {
 		// something went terribly wrong
+		document.write('<img src="http://logs-01.loggly.com/inputs/88a88e56-77c6-49cc-af41-6f44f83fe7fe.gif?message=something%20went%20terribly%20wrong"/>');
```

Comment:
For now I'm just logging the same message that was in the comment above. I'm logging it to loggly because that is quickest and most reliable solution to implement this - if we find out that this problem really happens - then we will have bigger discussion what we should do about it. I'm coming from: any logging for it is better than no logging at all.